app/local.env

```diff
@@ -36,6 +36,11 @@ POSTGRES_PASSWORD=secret123
 # value specified in the docker-compose file
 DB_HOST=localhost
 
+# When an error occurs with a SQL query,
+# whether or not to hide the parameters which
+# could contain sensitive information.
+HIDE_SQL_PARAMETER_LOGS=TRUE
+
```

Comment:
This seems useful to include in the non-generic repo.

Reply:
Added to the cleanup PR and merged that, so it's upstream of this change now.